mono/utils/mono-dl.c

```diff
@@ -37,11 +37,11 @@ struct MonoDlFallbackHandler {
 static GSList *fallback_handlers;
 
 
-/*                                                                                                                                                                                                
```

Comment:
It seems our formatting tool removed trailing whitespace on a number of lines in this file. I think this is generally a good thing, but I did not change this line or any of them below in any meaningful way. Should I put back the trailing whitespace so as not make make the diff more confusing?